### DISCUSSION TITLE
The dart eval package made it possible to use external sources using the dart language

### DISCUSSION THREAD

**Discussion by kodjodevf** [kodjodevf/mangayomi#41] [General]
At the start of the project all the extensions were hardcoded, but at some point I started wondering how to create an external source, I was on several Reddit forums etc. but finally I came across the wonderful package [dart eval](https://github.com/ethanblake4/dart_eval) (A flexible Dart bytecode compiler and interpreter written in Dart), it's really easy to understand, and the project is only growing and thanks really to @ethanblake4 for making this project possible.

**Comment by ethanblake4** [comment]
Thank you, glad to hear it! Can I have your permission to possibly feature your app in the future on the dart_eval pub page as an app that uses dart_eval?

**Reply by kodjodevf** [reply]
Of course you have my full permission
